### PR TITLE
Update get_cmaize.cmake

### LINKED ISSUE #97: Use Tags for Dependencies with `FetchContent_Declare`
**Is your feature request related to a problem? Please describe.**
You introduced a breaking change with https://github.com/CMakePP/CMakeTest/commit/393a85e774a2c4cb89ac720abe3ab252213bf933 for all your current depndencies (CMaize).

**Describe the solution you'd like**
For all your dependencies you also need to pin to tags with the `GIT_TAG` attribute (see https://cmake.org/cmake/help/latest/module/FetchContent.html) to not introduce breaklng changes.

**Describe alternatives you've considered**
Forking your CMakeTest repro and fix the `FetchContent_Declare` usage by using tags.

**Additional context**
It is good practice to pin to a tag instead to the master branch for dependencies.

### PR DESCRIPTION
**Pull request by ryanmrichard**
**Is this pull request associated with an issue(s)?**
Fixes #97.

**Description**
At present we were not pinning the version of CMaize that CMakeTest used. This PR fixes that. 

**TODOs**
Better understand if this really fixes #97.

### CODE REVIEW

File: cmake/get_cmakepp_lang.cmake [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#[[
+# This function encapsulates the process of getting CMakePPLang using CMake's
+# FetchContent module. When CMaize supports find_or_build for CMake modules this
+# file will be deprecated.
+#]]
+function(get_cmakepp_lang)
+    include(
+        cmakepp_lang/cmakepp_lang
+        OPTIONAL
+        RESULT_VARIABLE cmakepp_lang_found
```
Comment (zachcran) [inline cmake/get_cmakepp_lang.cmake:34]: I never realized that there were extra possible kwargs for `include()`. Good to know!

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,21 +1,30 @@
...
-cmake_minimum_required(VERSION "${_ct_min_cmake_version}") #Required for FetchContent_MakeAvailable()
+#3.19 needed because of CMaize
+set(_ct_min_cmake_version "3.19" CACHE INTERNAL "")
+cmake_minimum_required(VERSION "${_ct_min_cmake_version}")
 project(CMakeTest VERSION 1.0.0 LANGUAGES NONE)
 
 if(NOT CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
-	set(_ct_min_cmake_version "${_ct_min_cmake_version}" PARENT_SCOPE) #Make sure projects using CMakeTest have this variable
+    #Make sure projects using CMakeTest have this variable
+	set(_ct_min_cmake_version "${_ct_min_cmake_version}" PARENT_SCOPE)
```
Comment (zachcran) [inline CMakeLists.txt:8]:
```suggestion
    set(_ct_min_cmake_version "${_ct_min_cmake_version}" PARENT_SCOPE)
```

File: CMakeLists.txt [hunk 3]
```diff
@@ -1,21 +1,30 @@
...
 endif()
 
+# Add cmake directory to CMake's module path, iff it's not already there
```
Comment (zachcran) [inline CMakeLists.txt:11]: Is the `iff` here a typo, or intended to mean "if and only if"?

File: CMakeLists.txt [hunk 4]
```diff
@@ -1,21 +1,30 @@
...
 list(FIND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake" path_has_cmake)
 if(${path_has_cmake} STREQUAL "-1")
     list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 endif()
+
+# TODO: Should this be in the above if block?
 set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}" CACHE STRING "" FORCE)
```
Comment (zachcran) [inline CMakeLists.txt:18]: I don't think it hurts for this to be outside of the above if block. At the worst, I think it is just writing the same `CMAKE_MODULE_PATH` that is already in the cache when the if statement is not triggered. You could probably save a few cycles if it was in the if statement, but not enough to really matter.

**Review by zachcran — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 3 above)
Comment (ryanmrichard) [reply CMakeLists.txt:11]: I meant it as the latter.

**Pull request merged**